docs/src/main/paradox/docs/delta/api/storages-api.md

```diff
@@ -37,23 +39,34 @@ local file-system structure and that Nexus has read and write access to the targ
   "readPermission": "{read_permission}",
   "writePermission": "{write_permission}",
   "capacity": "{capacity}",
-  "maxFileSize": {max_file_size}
+  "maxFileSize": {
```

Comment:
Is this on purpose ?

Reply:
Reformatted the whole file by mistake; diff on the file is legible now